.github/workflows/docker-build-publish.yml

```diff
@@ -1,5 +1,6 @@
 name: Docker Build & Publish
 
+# Trigger on push events to main (i.e. merges), pushing new semantic version tags and all PRs # yamllint disable-line rule:line-length
```

Comment:
[optional]

```suggestion
# Trigger on push events to main (i.e. merges), pushing new semantic version
# tags and all PRs
```

If you use VSCode, an extension can do this wrapping for you: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap

Reply:
Thank you @rootulp  :)
anyways, I have to fix the Yaml linter soon in order to also prevent this kind of issues, sometimes it's not a comment and we cannot change it to multiline 😓

Reply:
One can selectively disable the line length rule via:

```yaml
# yamllint disable-line rule:line-length
- This line is waaaaaaaaaay too long but yamllint will not report anything about it.
  This line will be checked by yamllint.
```

per https://yamllint.readthedocs.io/en/stable/disable_with_comments.html

Reply:
yes hehe I used this one, but at the end of the line instead of above of it :D 
thanks for the info! 😊